services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py

```diff
@@ -80,3 +83,40 @@ def create_job_from_study(
     )
 
     return new_job
+
+
+async def create_job_outputs_from_project_outputs(
```

Comment:
It would be nice to have a more elegant implementation of this one. But I am guessing you just copied it over for now.

Reply:
yes, for now is copy&paste. I will follow up. There are some problems with that implementation that I already noted. Will follow up :-)